src/main/java/com/ibm/crypto/plus/provider/XDHPrivateKeyImpl.java

```diff
@@ -523,6 +539,8 @@ public void destroy() throws DestroyFailedException {
             destroyed = true;
             if (this.key != null)
                 Arrays.fill(this.key, (byte) 0x00);
+            if (super.key != null)
+                Arrays.fill(this.key, (byte) 0x00);
```

Comment:
I think you mean `Arrays.fill(super.key, (byte) 0x00);`.

Reply:
Right, changed it.